### ISSUE TITLE
Magnetic field status

### ISSUE THREAD

**Issue by Maasik** [S-LABc/MT6701-Arduino-Library#3]
Здравствуйте.
Не могли бы добавить чтение "Magnetic field status Mg[3:0]"?

**Comment by S-LABc** [comment]
Приветствую! Да, добавлю в ближайшее время. Скопилось некоторое количество правок для библиотеки

**Comment by S-LABc** [comment]
> Здравствуйте. Не могли бы добавить чтение "Magnetic field status Mg[3:0]"?

Приветствую! Да, добавлю в ближайшее время. Скопилось некоторое количество правок для библиотеки

**Comment by Maasik** [comment]
Отлично, буду ждать.
Спасибо!

**Comment by Maasik** [comment]
Есть продвижение?

**Comment by S-LABc** [comment]
> Есть продвижение?

Да. По I2C шине не описаны регистры для получения таких данных. Вряд ли стану продолжать копать в этом направлении. На мой взгляд датчик и документация к нему очень слабые.

Как альтернатива - AS5600 и AS5601
